Directory.Packages.props

```diff
@@ -341,17 +341,61 @@
     <PackageVersion Include="Moq"                                                             Version="4.18.4"          />
     <PackageVersion Include="System.Linq.Async"                                               Version="6.0.1"           />
 
+    <!--
+      Note: OpenIddict uses PolySharp to dynamically generate polyfills for types that are not available on
+      some of the targeted TFMs (e.g Index, Range or nullable attributes on .NET Framework/.NET Standard).
+    -->
+    <GlobalPackageReference Include="PolySharp" Condition=" '$(DisablePolySharp)' != 'true' " Version="1.13.1"          />
+  </ItemGroup>
+
+  <!--
+                                          ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                          █████ ▀██ ██ ▄▄▄█▄▄ ▄▄███▀▄▄▀████ ▄▄ ██
+                                          █▀▀██ █ █ ██ ▄▄▄███ █████▀▄▄▀█▀▀█ ▀▄ ██
+                                          █▄▄██ ██▄ ██ ▀▀▀███ █████▄▀▀▄█▄▄█ ▀▀ ██
+                                          ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
```

Comment:
This is a place where I wouldn't expect to find ascii art 😅

Reply:
ASCII art FTW! ❤️ 

(wait till you see https://github.com/openiddict/openiddict-core/blob/dev/src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml 🤣)

Reply:
That's awesome 💯

Reply:
On a more serious note, OpenIddict's `Directory.Packages.props` is quite special, as package versions are defined per-TFM (so that `net6.0` references `Microsoft.Extensions.*` 6.0 for instance): using ASCII art makes identifying the per-TFM sections easier 😃